python/tests/test_drawing.py

```diff
@@ -1765,13 +1771,24 @@ def test_draw_simple_ts(self):
         self.verify_basic_svg(svg, width=200 * ts.num_trees)
 
     def test_draw_integer_breaks_ts(self):
-        r_map = msprime.RecombinationMap.uniform_map(1000, 0.001, num_loci=1000)
-        r_map = msprime.RecombinationMap.uniform_map(1000, 0.005, num_loci=1000)
-        ts = msprime.simulate(5, recombination_map=r_map, random_seed=1)
+        try:
+            recomb_map = msprime.RecombinationMap.uniform_map(length=1000, rate=0.005)
+            ts = msprime.simulate(
+                5, recombination_map=recomb_map, discrete_genome=True, random_seed=1
+            )
+        except TypeError:
+            # Fallback for older versions of msprime, which use num_loci
+            recomb_map = msprime.RecombinationMap.uniform_map(
+                length=1000, rate=0.005, num_loci=1000
+            )
+            ts = msprime.simulate(5, recombination_map=recomb_map, random_seed=1)
```

Comment:
In test code I think it is better if we keep things simple and only support the latest, released version of msprime. @jeromekelleher might disagree?

Reply:
> Great stuff - just a couple of comments. Am I right in thinking `symbol_size` is an argument because the radius can't be done in CSS? Sorry if I missed a discussion on that.

Radius can't be done in SVG 1.1 (it can be set using CSS in SVG 2). Previously I (inconsistently) hard coded the square to be 6x6 px, and the circle to be radius 1, then overwrote the radius to 3px using css (which didn't work on Inkscape). it seemed better to hard code a default symbol size somewhere, and use it for both the circle and the square. So I made it user-configurable, but just not documented (so that in extremis, Inkscape users could use it without having to go via the chromium route).

Reply:
> In test code I think it is better if we keep things simple and only support the latest, released version of msprime. @jeromekelleher might disagree?

I suspect [he agrees](https://github.com/tskit-dev/tsinfer/pull/333#issuecomment-678973542). but will it not be a pain that CI for tskit will start failing as soon as msprime 1.0 is released to Conda ?

Reply:
We would pin the version in CI, then PR the needed changes. No biggie.

Reply:
Done in https://github.com/tskit-dev/tskit/pull/790/commits/9a7213d2818371e9f56ab07c616ccc91b21ef864